test/api/publish.test.js

```diff
@@ -128,24 +130,18 @@ describe('publishing endpoints', () => {
     );
   });
 
-  it('returns an error when the sent data is bigger than the length', async () => {
-    jest.spyOn(process.stdout, 'write').mockImplementation(() => {});
-    const req = getPreconfiguredRequest();
-    await req.set('Content-Length', String(3)).send('aaaa').expect(400); // 400 means Bad Request. It's generated automatically by Koa.
-    expect(process.stdout.write).toHaveBeenCalledWith(
-      expect.stringContaining('server_error')
-    );
-  });
```

Comment:
I remove this test case because:
* I couldn't get the error I wanted with the changes
* this is auto-handled by node and koa -- when I tested, I got results like 400, 500 or 405 (method not allowed). The underlying node error is actually "invalid method" (see https://github.com/nodejs/node/issues/33728 for more explanation about that)
* this is quite an edge case so I didn't feel it was important to invest time to make a test for it after all.